src/frontend/src/Components/map/dataLoaders/events.js

```diff
@@ -1,16 +1,36 @@
-import { compareRoutePoints } from '../helpers/spatial';
 import { getEvents } from '../../data/events';
 
 // Event layers
-export const loadEvents = async (route, events, filteredEvents, eventFilterPoints, dispatch, displayError, worker) => {
-  const routePoints = route ? route.points : null;
+export const loadEvents = async (route, events, filteredEvents, eventFilterPoints, dispatch, displayError, worker, isInitialLoad = true, trackedEventsRef) => {
+  // Fetch data
+  const eventData = await getEvents().catch((error) => displayError(error));
 
-  // Load if filtered objs don't exist or route doesn't match
-  if (!filteredEvents || !compareRoutePoints(routePoints, eventFilterPoints)) {
-    // Fetch data if it doesn't already exist
-    const eventData = events ? events : await getEvents().catch((error) => displayError(error));
```

Comment:
Removed if block. Previously we only fetched data on initial load and route update, but now that we have polling this doesn't make sense any more.